teamcity: failed tests on master: roachtest/acceptance

The following tests appear to have failed:

[#905495](https://teamcity.cockroachdb.com/viewLog.html?buildId=905495):


```
--- FAIL: roachtest/acceptance/cli/node-status (27.813s)
	test.go:503,cli.go:79,cli.go:85,acceptance.go:59: expected [is_available is_live true true false false true true], but found [is_available is_live true true true true false false]:

--- FAIL: roachtest/acceptance/cli/node-status (27.813s)
	test.go:503,cli.go:79,cli.go:85,acceptance.go:59: expected [is_available is_live true true false false true true], but found [is_available is_live true true true true false false]:


```

Please assign, take a look and update the issue accordingly.

--- a/pkg/cmd/roachtest/cli.go
+++ b/pkg/cmd/roachtest/cli.go
@@ -24,7 +24,7 @@ import (
 
 func runCLINodeStatus(ctx context.Context, t *test, c *cluster) {
 	c.Put(ctx, cockroach, "./cockroach")
-	c.Start(ctx, c.Range(1, 3))
+	c.Start(ctx, c.Range(1, 3), startArgs("--sequential"))
 
 	db := c.Conn(ctx, 1)
 	defer db.Close()

